docs/_files/input_parameters_list.csv

```diff
@@ -1,32 +1,32 @@
+Interest rate,Zinsrate,Annahme der Zinsrate w�hrend der Projektlaufzeit,A guess of what the  interest rate will be in during the project lifetime. Could be provided as constant or timeseries.,EUR,Value provided,float,Financial,,project,,,function exists,,Estimate the project costs
+Costs of replacement,Wiederbeschaffungskosten,Wiederbeschaffungskosten/ Kosten f�r die Verbesserung der aktuellen Systemkomponenten/Technologien,Costs of replacement/improvement of the current system components/assets,EUR,Value provided for each accepted asset type,float,Financial,,component,,,"not sure, if this exists already, but I think if not it's easy to implement; the mobility team has this in their tools as far as I remember",,
+Grid stability fee,Netzentgelt,"Entgelt, als Beitrag zur Netzstabilit�t",Fee/payment for contribution to grid stability,EUR,Value provided,float,Financial,,scenario,,,fix cost?,,
+Spatial resolution,r�umliche Aufl�sung,"Gew�nschte r�umliche Aufl�sung (z.B. Geb�ude, Quartiere)",Define the wished spatial resolution (it is wished that it can range from building up to districts),,,,GIS,,hidden,,,spatial resolution is flexible, it's always a matter of data ,
+New power plant available space,verf�gbare Potentialfl�che,Verf�gbare Fl�chen f�r den Zubau von Erzeugereinheiten,Available space for new power plants (for PV installation for example),,,float,GIS,,component,,,upper bounds exist,,"Space on the rooftop is only for solar, fields can be solar and wind, inside existing building diesel/gas/heat exchanger can be installed but no solar or wind"
+Consumer requirements,Anforderungen der Energieverbraucher ,Anforderungen der Energieverbraucher (z.B.. gew�nschter Lokalisierung),consumer requirements such as a specific localisation and wished minimal load profile (e.g. in an industry park),,,,GIS/Technical,,constraint,,OP_IN_27,?,,
+Existing grid structure (v2.0 of the tool),existierende Netzstruktur,GPS Koordinaten der existierenden Netzinfrastruktur,GPS coordinates of current grid structure (lines and hubs),,,,GIS/Technical,,hidden,,,"in the Ego-Tools I guess, but this is a topic I am not familiar with",,
+Future grid structure (v2.0 of the tool),zuk�nftige Netzstruktur,GPS Koordinaten der zuk�nftigen Netzinfrastruktur,GPS coordinates of future grid structure (lines and hubs),,,,GIS/Technical,,hidden,,,"in the Ego-Tools I guess, but this is a topic I am not familiar with",,
+Security of supply,Versorgungssicherheit,"Welche Technologien existieren, um die Versorgungssicherheit zu gew�hrleisten (z.B.  Notstromaggregat)", e.g. existence of back-up system,,,,Technical,,hidden,,,there are many components and constraints for a flexible backup system,,
+Balancing ratio for anciliary services,?,?,Balancing ratio for ancilliary services (need a more precise definition as PF could not re-figure out what this means),,,array,Technical,,hidden,,,?,,
```

Comment:
I have no idea what is meant with this parameter_name. Can you help out @Bachibouzouk ?

Reply:
@FHell how would you translate "Balancing ratio for anciliary services" in german?